support/rally.js

```diff
@@ -152,6 +175,15 @@ export default class Rally {
     }
   }
 
+  /**
+   * A listener which fires when the study is paused or resumed.
+   *
+   * @param {function} callback
+   */
+  addStateListener(callback) {
+    this._listener = callback;
+  }
+
```

Comment:
We should make this a mandatory parameter of `initialize`, otherwise folks will forget to call it :)

Reply:
> We should make this a mandatory parameter of `initialize`, otherwise folks will forget to call it :)

Hm, do you mean we'd pass the callback function as part of initialize instead of the rally study needing to call `addStateListener()`?

Reply:
Yes!